src/pages/GamePage.tsx

```diff
@@ -29,12 +29,23 @@ const GamePage: React.FunctionComponent = () => {
     }, []);
 
     return (
+      
       <div className='flex flex-col justify-center gap-8 h-dvh'>
         <div className='flex flex-col justify-center items-center gap-4'>
           <GameTitle></GameTitle>
-          <div className=' border border-white/5 bg-slate-800/25 rounded-xl p-6 items-center flex flex-col gap-2'>
-            <SelectedImage source={selectedCat}></SelectedImage>
-            <SelectedPhrase phrase={selectedPhrase}></SelectedPhrase>
+          <div className=' border border-white/5 bg-slate-800/25 rounded-xl p-6 text-center items-center'>
+            {/* Using AND operator to avoid the broken image void that would be displayed if the user selected a phrase before an image*/}
```

Comment:
I think this comment can be removed. Its up to you whether you want it or not

Reply:
In retrospect, I think it might a little bit self-evident hehe
I'll remove it.